src/index.js

```diff
@@ -90,17 +93,20 @@ function createSolutionInfos() {
   ));
 }
 
-function processNext(next, config) {
-  const branch = config.submodules[next];
-  console.log(`Processing ${next}`);
+function processSubmodule(name, config) {
+  const branch = config.submodules[name];
 
-  const [owner, repo] = next.split('/');
+  const [owner, repo] = name.split('/');
   const submodule = {
+    _logs: [],
     owner,
     repo,
     branch,
-    remoteUrl: `git@git:${owner}/${repo}.git`
+    remoteUrl: `git@git:${owner}/${repo}.git`,
+    log(message, indent=0) { this._logs = this._logs.concat(message.split(/\r?\n/).map((line) => { return Array(indent + 1).join(' ') + line; })); },
```

Comment:
Yes, I tried it with babel-node and also built the project with gulp and it produces ' '.repeat(indent) + line, but when you try to run it in node v0.12.7, you get `undefined is not a function`.

I think we have to wait for a future version of node or run node with `--harmony`.

Changing the first line of dist/index.js to:

    #!/usr/bin/env node --harmony

allows repeat() to work.  I will figure out how to get gulp to add the flag.